include/deal.II/fe/fe_nedelec.templates.h

```diff
@@ -0,0 +1,103 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2002 - 2025 by the deal.II authors
```

Comment:
Let's assume that this is essentially new code:

```suggestion
// Copyright (C) 2025 by the deal.II authors
```